test-definitions-openapi/fern/apis/code-samples-open-api/.definition/api.yml

```diff
@@ -0,0 +1,8 @@
+name: api
+error-discrimination:
```

Comment:
@omarrida can you make sure to not commit the `.definition` folders? maybe you can just add to `.gitignore`